source/gloperate-text/source/stages/GlyphRenderStage.cpp

```diff
@@ -48,7 +48,7 @@ void GlyphRenderStage::onProcess(gloperate::AbstractGLContext *)
 
     if (!fbo)
     {
-        fbo = globjects::Framebuffer::defaultFBO();
+        fbo = globjects::Framebuffer::defaultFBO().get();

```

Comment:
This returns a `unique_ptr<Framebuffer>` which is immediatly deleted, as the `unique_ptr`  is never saved